src/engraving/api/v1/qmlpluginapi.h

```diff
@@ -121,6 +121,7 @@ class PluginAPI : public QQuickItem, public muse::extensions::apiv1::IPluginApiV
     /** List of currently open scores (read only).\n \since MuseScore 3.2 */
     Q_PROPERTY(QQmlListProperty<mu::engraving::apiv1::Score> scores READ scores)
 
+public:
```

Comment:
I don't see why this one is necessary

Reply:
At the moment, enum propties implementations (declared by DECLARE_API_ENUM) used here https://github.com/musescore/MuseScore/blob/master/src/engraving/api/v1/engravingapiv1.h#L91

Reply:
Aha, now I understand it: previously they were already public as a side effect of the INJECT macros above it (I didn't see those earlier). So yes, this makes sense!